docs/source/_includes/consensus-node-reqs.inc

```diff
@@ -0,0 +1,18 @@
+  - :term:`PBFT consensus` requires four or more nodes. At least four nodes must
+    be configured and running in order for the network to start.
+
+  - :term:`PoET consensus` requires three or more nodes. You can start the first
+    node and test basic functionality, then add the other nodes.
+
+    .. note::
+
+       This procedure uses PoET simulator consensus (also called PoET CFT
+       because it is crash fault tolerant), which is a version of PoET-SGX
+       consensus that runs on any processor.
+
+  - :term:`Devmode consensus <dev mode consensus>` has no minimum requirement,
+    but it is not recommended for a multiple-node or production networks.
+    Devmode is a light-weight consensus that is intended for short-term testing
+    on a single node or a very small network (two or three nodes). It does not
+    handle forks and is not crash fault tolerant.
```

Comment:
Technically, it does handle forks, just in a very inefficient way.

Reply:
Rewrote to clarify.